tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "target": "es5",
     "baseUrl": ".",
     "module": "commonjs",
-    "outDir": "./build",
+    "outDir": "./src/build",
```

Comment:
Is the `build` folder nested inside `src`? I think next to each other should be better, but I might overlook something

Reply:
ya now `build` folder will be inside `src`, when we run `npm pack` or `yarn pack`, it will produce the same tgz file as before this PR.

Reply:
the idea is for now we produce the tgz file with the same content as before this PR. Then in the next major release, we can consider to get rid of `build` in the tgz file content. So the whole tgz file content will potential be:
```
node_modules
-- jest-preset-angular
    -- index.js
    -- jest-preset.js
    -- zone-patch.js
    -- <other_transformers>.js
    -- <other_snapshot_serializers>.js
```